src/main/java/com/epam/model/Person.java

```diff
@@ -10,4 +10,10 @@
     private String email;
     private String password;
     private PersonRoleEnum role;
+
+    public Person(String email, String password, String role) {
```

Comment:
You can try allArgsConstructor here

Reply:
The thing is that we should create an instance of Person in Controller. In all-args construct we must transfer to construct id, but we wont use it in any of other methods.

Reply:
Done by @RequairedArgsConstructor